pkg/services/login/authinfo.go

```diff
@@ -36,10 +36,13 @@ type Store interface {
 
 const (
 	// modules
+	PasswordAuthModule  = "password"
+	APIKeyAuthModule    = "apikey"
```

Comment:
also supernit: not sure how much it matters, but it would be good to have some consistency on naming. Should it be `apikey`, `api.key` or `api_key`? We seem to use all of them already 🙈

Reply:
I think you're right that we need to leave the existing ones as they are. I think as long as we stick with one convention for the future auth modules it's fine.